documentation/change-log.md

```diff
@@ -0,0 +1,98 @@
+OP-TEE - version 0.1.0
+======================
+
+New features
+------------
+Below is a summary of the most important features added, but at the end you will
+find a link that present you all commits between the current and previous
+release tag.
+
+*   GlobalPlatform Client API v1.0 support.
+
+*   GlobalPlatform Internal API v1.0 support.
+
+*   GlobalPlatform Secure Elements v1.0 support.
+
+*   Add hardware support for
+
+    *   Allwinner A80, ARMv7-A.
+
+    *   ARM Juno Board, ARMv8-A.
+
+    *   Foundation Models, ARMv8-A.
+
+    *   Fast Models, ARMv8-A.
+
+    *   QEMU, ARMv7-A.
+
+    *   STM Cannes, ARMv7-A.
+
+    *   STM Orly2, ARMv7-A.
+
+*   Added LibTomCrypt as the default software cryptographic library.
+
+*   Added cryptographic abstraction layer in on secure side to ease the use of
+    other cryptographic software libraries or adding support for hardware
+    acceleration.
+
+*   Extended cryptographic API with support for HKDF, Concat KDF and PBKDF2.
+
+*   SHA-1 and SHA-256 ARMv8-A crypto extension implementation.
+
+*   Enabled paging support in OP-TEE OS.
+
+*   Added support for xtest (both standard and extended) in QEMU and FVP setup
+    scripts.
+
+*   Add documentation for the os design, cryptographic abstraction layer, secure
```

Comment:
I would suggest CHANGELOG.md which seems the most common name (and also consistet with README.md).